kernel/test-helper/pom.xml

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <parent>
+    <artifactId>kernel</artifactId>
+    <groupId>org.sakaiproject</groupId>
+    <version>26-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.sakaiproject.kernel</groupId>
+  <artifactId>sakai-test-helper</artifactId>
+  <packaging>jar</packaging>
+  <name>Sakai Kernel Test Helper</name>
```

Comment:
There may be a way to make this jar without replicating all of the stuff from kernel or deploy that makes it all work, but there are a bunch of places where we change scope from provided to compile so they get unrolled and flattened. This is the painstakingly created minimal set needed to boot the kernel.